modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -97,4 +97,124 @@ Status description:: error: connection exception - alias chains are not permitte
+
+=== 22N04 
+
+Status description:: error: data exception - invalid input value. Invalid input { $input } for { $context }. Expected one of { $inputOredList }.
+
+=== 22N05
+
+Status description:: error: data exception - input failed validation. Invalid input { $input } for { $context }.
+
+=== 22N06 
+
+Status description:: error: data exception - empty input string. Invalid input. { $option } needs to be specified.
+
+=== 22N07 
+
+Status description:: error: data exception - invalid pre-parser option key. Invalid pre-parser option(s): { $optionList }.
+
+=== 22N08 
+
+Status description:: error: data exception - invalid pre-parser combination. Invalid pre-parser option, cannot combine { $option1 } with { $option2 }.
+
+=== 22N09 
+
+Status description:: error: data exception - conflicting pre-parser combination. Invalid pre-parser option, cannot specify multiple conflicting values for { $option }.
+
+=== 22N10 
+
+Status description:: error: data exception - invalid pre-parser option value. Invalid pre-parser option, specified { $input } is not valid for option { $option }. Valid options are: { $optionAndedList }.
+
+=== 22N11 
+
+Status description:: error: data exception - invalid argument. Invalid argument: cannot process { $input }.
+
+=== 22N12 
+
+Status description:: error: data exception - invalid date, time, or datetime format. Invalid argument: cannot process { $input }.
+
+=== 22N13 
+
+Status description:: error: data exception - invalid time zone. Specified time zones must include a date component.
+
+=== 22N14 
+
+Status description:: error: data exception - invalid temporal value combination. Cannot select both { $temporal } and { $component }.
+
+=== 22N15 
+
+Status description:: error: data exception - invalid temporal component. Cannot read the specified { $component } component from { $temporal }.
+
+=== 22N16 
+
+Status description:: error: data exception - invalid import value. Importing entity values to a graph with a USE clause is not supported. Attempted to import { $expr } to { $graph }.
+
+=== 22N17 
+
+Status description:: error: data exception - invalid date, time, or datetime function field name. Cannot read the specified { $component } component from { $temporal }.
+
+=== 22N18 
+
+Status description:: error: data exception - incomplete spatial value. A { $crs } POINT must contain { $mapKeyAndedList }.
+
+=== 22N19 
+
+Status description:: error: data exception - invalid spatial value. A POINT must contain either 'x' and 'y', or 'latitude' and 'longitude'.
+
+=== 22N20 
+
+Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with { $crs } coordinate reference system (CRS) and { $coordinates } coordinates. Use the equivalent { $crs2 } coordinate reference system instead.
```

Comment:
```suggestion
Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with { $dim1 }D coordinate reference system (CRS) and { $value } coordinates. Use the equivalent { $dim2 }D coordinate reference system instead.
```

We noticed that the parameters were bad here, so we have updated it in the code. But seems like I missed to update the spreadsheet, have done that now.